### PR TITLE
Python Built-in-function : .dir()

### PR DESCRIPTION
**Pull request by VasilisCbt17gr**
### Description

- Added the dir() bult-in-function for Python
- Closes Issue #2137 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks good to me! 😊👍🏾

**Comment by VasilisCbt17gr** [comment]
@CBID2  Could you check it please?

**Comment by SSwiniarski** [comment]
@VasilisCbt17gr You already have an outstanding PR on this issue that's currently being reviewed, [#2303](https://github.com/Codecademy/docs/pull/2303). Why are you submitting another one?

**Comment by VasilisCbt17gr** [comment]
@SSwiniarski  I'm training on gitlab and i couldn't complete  issue #2303 . So, i did the procedure from scratch ,reviewed each video carefully and submitted it again.

**Comment by VasilisCbt17gr** [comment]
@cbid2 I did every check using yarn successfully. But there is still a sign "Required" and a message " 1 workflow awaiting approval ".
What should i do to proceed? Thank you.

**Comment by VasilisCbt17gr** [comment]
@CBID2  So, have i wait or make some changes?

**Comment by CBID2** [comment]
> @CBID2 So, have i wait or make some changes?

You'd have to wait for the maintainers, the people I tagged, @VasilisCbt17gr. Also, don't forget to update your branch.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@VasilisCbt17gr I did a first review. Most important is your PR touches two files aside from the entry. We cannot merge with those changes. So you need to undo those changes or submit a new PR without them.

**Comment by VasilisCbt17gr** [comment]
@SSwiniarski  Could you review the changes?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@VasilisCbt17gr You haven't addressed three of my comments. Most important are the changes to the two system files in this PR **this PR cannot be merged with those changes**. You need to undo those changes in this PR, or create a fresh branch off of main for a new PR that contains just your entry.

File: yarn.lock
Comment (SSwiniarski) [reply yarn.lock:1]: @VasilisCbt17gr The easiest way to fix this at this point is for you to make a fresh branch off of main, copy your entry there, and create a fresh PR. The yarn.lock and package.json files should not be touched in the PR, and these are heavily modified. We can't merge with those changes.

**Comment by VasilisCbt17gr** [comment]
@SSwiniarski  Could you check issue #2528 ? Thank you.

**Comment by SSwiniarski** [comment]
Closing because a new PR was created for this issue.

**Pull request closed without merging**
